armi/physics/neutronics/crossSectionSettings.py

```diff
@@ -503,6 +515,7 @@ def __init__(
         minDriverDensity=0.0,
         ductHeterogeneous=False,
         traceIsotopeThreshold=0.0,
+        xsTempIsotope="U238",
```

Comment:
Is this only for homogeneous compositions or does it also work with other modeling options as well?

Reply:
I'd guess it's independent of the treatment since this occurs in the XS group manager rather than specific modeling options applied for the input generation. Also, I assume that U238 is selected here because of fuel, but does this work if you select an isotope that isn't in the composition? For example if the isotope is not in the block, does this fail, should it fail, or should the environment/temperature group never be updated?

Reply:
works for other based my expected implementation and on Mikes testing.